pkg/batch/api/api_test.go

```diff
@@ -485,40 +446,6 @@ func TestProgressiveRolloutWatcher(t *testing.T) {
 	}, setupMock)
 }
 
-func TestDomainEventInformer(t *testing.T) {
```

Comment:
Since we moved `createPuller` to informer, we are unable to mock puller. Therefore, let's delete this unit test here.

We will need to create an end-to-end test for `DomainEventInformer`.